src/main/osd/osd.c

```diff
@@ -464,11 +467,13 @@ static bool isSomeStatEnabled(void)
 
 static void osdShowStats(uint16_t endBatteryVoltage)
 {
-    uint8_t top = 2;
+    uint8_t top = 1;    /* first fully visible line */
     char buff[OSD_ELEMENT_BUFFER_LENGTH];
 
     displayClearScreen(osdDisplayPort);
-    displayWrite(osdDisplayPort, 2, top++, "  --- STATS ---");
+
+    if (IS_DISPLAY_PAL)
```

Comment:
It would be much nicer to determine of the heading line can be displayed based on how many lines are available vs. how many are needed rather than always committing the heading for NTSC. `max7456GetRowsCount()` will return the available rows based on the current video mode. Then you could eliminate any PAL vs. NTSC logic.

Reply:
Unfortunately there's no easy way of determining how many lines will be actually shown... A combination of ifdefs/feature checks/and other checks would be needed to be repeated...
I'd rather not waste flash for this...

Reply:
the stats line could be a configurable stats item.

Reply:
I'd avoid this :)
It would only add complexity without any real value...